### DISCUSSION TITLE
An Issue With Creating & Publishing Multiple Quarto Sites

### DISCUSSION THREAD

**Discussion by theaeonwanderer** [quarto-dev/quarto-cli#3316] [Q&A] [answered]
When we publish a Quarto site from GitHub Pages, it takes the format `username.github.io`. I wanted to create and publish a site with a different name say `codewithme.github.io`. Somehow, the  `username.github.io` sticks with it publishing it as `username.github.io/codewithme.github.io` 😮!

Is there a way to resolve this or will I have to create a different GitHub account with my `codewithme` username 😏?

Thanks!

**Comment by mcanouil** [comment, marked as answer]
This is a GitHub constraint.
If you don't own the account, you won't be allowed to publish to it, because it would be like trying to impersonate some user or organisation account.
To note "username.github.io" is a special name of repository to publish to the root of "username.github.io" website.
All of your other repository published on GitHub Pages will be in the form "username.github.io/repository".
There is absolutely nothing you can do to change that.

If you really want to customise the subdomain and/or the apex, then buy a domain. 🤷‍♂️

**Reply by theaeonwanderer** [reply]
Thanks buddy!
